src/main/java/com/google/alpollo/SongInfoServlet.java

```diff
@@ -0,0 +1,45 @@
+package com.google.alpollo;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.alpollo.model.SongInfo;
+import com.google.gson.Gson;
+
+/** Servlet created to deal with songInfo related operations. */
```

Comment:
Please, enumerate these operations.

Reply:
Well each operation is described inside, above the doGet() method and the doPost() method.

Reply:
It's good that there's just 2 methods, but if they were more, it would be quite uncomfortable to scroll down through all of them((

Reply:
You can just add that this servlet `retrieves and saves songInfo object to the database`